modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -1,35 +1,37 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc
 
 [id="identity-provider-provisioning-user-lookup-mapping_{context}"]
 = Manually provisioning a user when using the lookup mapping method
 
 When using the `lookup` mapping method, user provisioning is done by an external system, via the API.
-Typically, identities are automatically mapped to users during login. The 'lookup' mapping method automatically
+Typically, identities are automatically mapped to users during login. The `lookup` mapping method automatically
 disables this automatic mapping, which requires you to provision users manually.
 
-
-.Procedure 
+.Procedure
 
 If you are using the `lookup` mapping method, use the following steps for each user after configuring
 the identity provider:
 
 . Create an {product-title} User, if not created already:
 +
+[source,terminal]
 ----
 $ oc create user <username>
 ----
 +
 For example, the following command creates an {product-title} User `bob`:
 +
+[source,terminal]
 ----
 $ oc create user bob
 ----
 
 . Create an {product-title} Identity, if not created already. Use the name of the identity provider and
 the name that uniquely represents this identity in the scope of the identity provider:
 +
+[source,terminal]
 ----
 $ oc create identity <identity-provider>:<user-id-from-identity-provider>
 ----
```

Comment:
Line 39 `The ``<identity-provider>`` is the name of the identity provider in the master configuration` mentions `master configuration`. `master configuration` is a master config file on master node, it only exists in OpenShift v3.x, not exists in OpenShift v4.x. You can remove the ` in the master configuration` words.